firewall/interception/windowskext/kext.go

```diff
@@ -170,19 +121,22 @@ func RecvVerdictRequest() (*VerdictRequest, error) {
 		time.Sleep(100 * time.Microsecond)
 	}
 
-	// timestamp := time.Now()
-	rc, _, lastErr := kext.recvVerdictRequest.Call(
-		uintptr(unsafe.Pointer(new)),
-	)
-	// log.Tracef("winkext: getting verdict request took %s", time.Now().Sub(timestamp))
+	timestamp := time.Now()
```

Comment:
I would recommend using the following pattern for this kind of debugging:
```
started := time.Now()
defer log.Tracef("winkext: getting verdict request took %s", time.Since(started))
```
This keeps everything together.